portfolio/src/main/java/com/google/sps/servlets/HelloWorldServlet.java

```diff
@@ -5,15 +5,36 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
 
 /** Handles requests sent to the /hello URL. Try running a server and navigating to /hello! */
 @WebServlet("/hello")
 public class HelloWorldServlet extends HttpServlet {
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello there, Im Edson!</h1>");
+    ArrayList<String> thoughs = new ArrayList<String>();
+    thoughs.add("Everyone can fly, the problem is that we're not crazy enough for it!");
+    thoughs.add("Connecting devices, enable them to be smarter and less vulnerable");
+    thoughs.add("If you see what almost no one sees, you should do what almost no one does!");
+
+    String toJson = convertToJson(thoughs);
+    response.setContentType("application/json;");
+    response.getWriter().println(toJson);
+  }
+
+  private String convertToJson(ArrayList<String> thoughs) {
+    String json = "{";
+    json += "\"though1\": ";
```

Comment:
Instead of creating a JSON object, you could create a JSON array, and then use a `for` loop to add all of the elements from your ArrayList to it. Up to you though!